environment/integration/helpers.go

```diff
@@ -34,6 +36,40 @@ func init() {
 	})))
 }
 
+// createTestTempDir creates a temporary directory with a shorter path on Windows
+func createTestTempDir(t *testing.T, prefix string) string {
+	var dir string
+	var err error
+
+	// Get test name and replace slashes with underscores to avoid path separator issues
+	testName := t.Name()
+	testName = strings.ReplaceAll(testName, "/", "_")
+
+	if runtime.GOOS == "windows" {
+		// On Windows, use a shorter base path to avoid path length issues
+		// Generate a unique short name using last 8 chars of test name
+		if len(testName) > 8 {
+			testName = testName[len(testName)-8:]
```

Comment:
🐛 **Bug (Low):** Test name truncation could cause collisions. Consider using a hash instead of just the last 8 characters

```suggestion
        if len(testName) > 8 {
            h := sha1.Sum([]byte(testName))
            testName = fmt.Sprintf("%.8x", h)
        }
```